webapp/package.json

```
@@ -1,41 +1,59 @@
 {
-  "name": "vuejs3",
-  "description": "SDF Todo list with vue cli and bootstrap.",
+  "name": "sdaf_todo_list",
+  "description": "SDAF Todo list with vue cli, bootstrap, ESLint, Travis & Apollo GraphQL.",
   "version": "1.0.0",
   "author": "Youri Seichter <yseichter@gmail.com>",
   "private": true,
   "scripts": {
-    "dev": "cross-env NODE_ENV=development webpack-dev-server --open --hot",
-    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules",
-    "test": "jest",
-    "lint": "./node_modules/.bin/eslint . ./**/*.vue"
+    "dev": "./node_modules/.bin/cross-env NODE_ENV=development webpack-dev-server --open --hot",
+    "build": "./node_modules/.bin/cross-env NODE_ENV=production webpack --progress --hide-modules",
```

A call of `npm run` will automatically look into `./node_modules/.bin` to resolve a binary. That's the reason why you use it after all!

So:
```suggestion
    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules",
```